Source/JavaScriptCore/wasm/js/JSWebAssembly.cpp

```diff
@@ -51,8 +51,9 @@ namespace JSC {
 
 STATIC_ASSERT_IS_TRIVIALLY_DESTRUCTIBLE(JSWebAssembly);
 
+// Uses UNUSED_FUNCTION because some constructors, e.g., for Arrays are currently not exposed.
 #define DEFINE_CALLBACK_FOR_CONSTRUCTOR(capitalName, lowerName, properName, instanceType, jsName, prototypeBase, featureFlag) \
-static JSValue create##capitalName(VM&, JSObject* object) \
+static UNUSED_FUNCTION JSValue create##capitalName(VM&, JSObject* object) \
```

Comment:
Note: I had to add this change as the Array constructor is defined and used in this macro, but is unused because I'm intentionally not exposing it as `WebAssembly.Array` in the JS Wasm namespace at this point. It's possible the JS API will include this later, but it might not.